includes/class-wc-stripe-webhook-state.php

```diff
@@ -0,0 +1,242 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WC_Stripe_Webhook_State.
+ *
+ * Tracks the most recent successful and unsuccessful webhooks in test and live modes.
+ * @since 4.4.2
```

Comment:
> Shouldn't it be @since 4.9.X... are we replacing them automatically during release?

Unfortunately not yet, that wouldn't work.

But I need to update it to 5.0.0 now.